app/controllers/event_steps_controller.rb

```diff
@@ -1,9 +1,11 @@
 class EventStepsController < ApplicationController
-  before_action :load_event
+  before_action :load_event 
 
   include WizardSteps
   self.wizard_class = Events::Wizard
 
+  before_action :set_page_title
```

Comment:
I think the variables this relies upon wont be available on every action in the controller, I think you'll need to `except` some of the actions - see

https://github.com/DFE-Digital/get-into-teaching-app/blob/master/app/controllers/concerns/wizard_steps.rb#L6